core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -36,7 +36,7 @@ import kafka.common.{KafkaException, TopicAndPartition}
 import collection.Set
 import collection.JavaConverters._
 
-class ControllerChannelManager(controllerContext: ControllerContext, config: KafkaConfig, time: Time, metrics: Metrics) extends Logging {
+class ControllerChannelManager(controllerContext: ControllerContext, config: KafkaConfig, time: Time, metrics: Metrics, testName: Option[String] = None) extends Logging {
```

Comment:
Passing a parameter called `testName` to a non test class is not ideal. We could either override the thread creation in the test or pass a more generic parameter (maybe requestThreadNamePrefix). What do you think?